usagov/vote-gov
# Vote-110 -> PT Accessibility Page

Ticket: [Vote-110](https://cm-jira.usa.gov/browse/VOTE-110)
Purpose: Add accessibility page for pt
Testing:

1. pull down branch 
2. run `npm run start`
3. visit pt translation and check that the new accessibility page loads and there are no major regressions
